Dockerfile

```diff
@@ -10,15 +10,15 @@ RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement
 
 # Build stage: Install yarn dependencies
 # ===
-FROM node:12 AS yarn-dependencies
+FROM node:16 AS yarn-dependencies
 WORKDIR /srv
 COPY . .
 RUN --mount=type=cache,target=/usr/local/share/.cache/yarn yarn install
 
 
 # Build stage:
 # ===
-FROM ubuntu:bionic AS build-documentation
+FROM ubuntu:focal AS build-documentation
```

Comment:
Should we give `impish` a try? See if things work. I think we should keep things as up to date as possible.

Reply:
It looks like `impish` is never going to be an LTS, so no need to think about it.